presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -32,18 +34,26 @@
     private final String catalogName;
     private final String schemaName;
     private final String tableName;
+    private final TupleDomain<ColumnHandle> tupleDomain;
+
+    public JdbcTableHandle(SchemaTableName schemaTableName, @Nullable String catalogName, @Nullable String schemaName, String tableName)
+    {
+        this(schemaTableName, catalogName, schemaName, tableName, TupleDomain.all());
```

Comment:
Do we need 2 ctors?

Reply:
We don't *need* it, but I tried originally without it and adding the extra args everywhere seemed worse than adding another constructor.